python/ray/train/tests/test_examples.py

```diff
@@ -88,24 +88,24 @@ def test_tensorflow_mnist_fail(ray_start_2_cpus):
 
 
 @pytest.mark.parametrize("num_workers", [1, 2])
-def test_torch_linear(ray_start_2_cpus, num_workers):
+def test_torch_linear(ray_start_4_cpus, num_workers):
     num_workers = num_workers
     epochs = 3
 
-    trainer = Trainer("torch", num_workers=num_workers)
     config = {"lr": 1e-2, "hidden_size": 1, "batch_size": 4, "epochs": epochs}
-    trainer.start()
-    results = trainer.run(linear_train_func, config)
-    trainer.shutdown()
-
-    assert len(results) == num_workers
+    trainer = TorchTrainer(
+        linear_train_func,
+        train_loop_config=config,
+        scaling_config=dict(num_workers=num_workers),
+    )
+    results = trainer.fit()
 
-    for result in results:
-        assert len(result) == epochs
-        assert result[-1]["loss"] < result[0]["loss"]
```

Comment:
Same here- can we keep these assert statements?

Reply:
added back in